sdk/src/tx/utils.ts

```diff
@@ -8,7 +8,7 @@ const COMPUTE_UNITS_DEFAULT = 200_000;
 
 export function wrapInTx(
 	instruction: TransactionInstruction,
-	computeUnits = 600_000 // TODO, requires less code change
+	computeUnits = 800_000 // TODO, requires less code change
```

Comment:
this is worrying lol which tx was failing?

Reply:
oh think was before merging master